containers/ecr-viewer/integration/core_database_repo.test.ts

```diff
@@ -70,68 +81,137 @@ describe("database_repo", () => {
         date_created: new Date("2025-01-01"),
         report_date: new Date("2025-02-07"),
       });
+      const actual = await database_repo.findEcrById("54321");
+      expect(actual).toEqual({
+        eICR_ID: "54321",
+        set_id: "setid",
+        data_source: "DB",
+        fhir_reference_link: "link",
+        eicr_version_number: "50000",
+        patient_name_first: "Boba",
+        patient_name_last: "Fett",
+        patient_birth_date: new Date("1969-02-10T05:00:00.000Z"),
+        date_created: new Date("2025-01-01"),
+        report_date: new Date("2025-02-06T05:00:00.000Z"),
+      });
     });
 
     it("should delete an ECR with a given id", async () => {
       await database_repo.deleteEcr("12345");
+      const actual = await database_repo.findEcrById("12345");
+      expect(actual).toBeUndefined();
     });
   });
 
   // ecr_rr_conditions
   describe("ecr_rr_conditions", () => {
+    var expected = {
```

Comment:
```suggestion
    const expected = {
```
`var` should almost never be used these days